README.md

```diff
@@ -9,88 +9,25 @@ Docker HUB url: https://hub.docker.com/r/dopplerrelay/doppler-relay-akamai-publi
 
 This image is ready to upload to akamai CDN the resource files. Contains the cdn-uploader script and his dependencies ready to run.
 
-Zero Step, define your environment variables:
+IMPORTANT: Environments variables defined with ENV in the dockerfile will be overrided by the host machine defined environment variables. This will be added in the README file.
```

Comment:
I think that this paragraph is not correct, because it depends on how it is implemented in host machine.

Maybe it is better to indicate that it is necessary to pass/define _AKAMAI_CDN_HOSTNAME_, _AKAMAI_CDN_USERNAME_, _AKAMAI_CDN_PASSWORD_, _AKAMAI_CDN_CPCODE_, _PROJECT_NAME_, _VERSION_NAME_ environment variables and _/source_ volume.